src/Auth/types.ts

```diff
@@ -0,0 +1,13 @@
+export type User = {
+    readonly id: number;
+    readonly email: string;
+    readonly name: string;
```

Comment:
- I can see there's `lastname` below, so it might be better to name it `firstname`, as for me it's less confusing
- I'd rather use `camelCase`